package.json

```diff
@@ -12,7 +12,7 @@
     "build:lib:umd:min": "talend-scripts build:lib:umd --prod",
     "start": "npm run storybook",
     "test": "talend-scripts test",
-    "storybook": "start-storybook -s static -p 6006",
+    "storybook": "BROWSER=none start-storybook -s static -p 6006",
```

Comment:
This is used to not open Storybook automatically https://github.com/storybookjs/storybook/issues/6201#issuecomment-812005603